docs/getting-started/install-configure.md

```diff
@@ -375,12 +375,12 @@ detailed instructions.
 ## Uninstall Provider
 ```
 kubectl delete -f provider.yaml
-kubectl delete secret -n crossplane-system --all
 ```
 
 ## Uninstall Crossplane
 ```
 helm delete crossplane --namespace crossplane-system
+kubectl delete secret -n crossplane-system --all
```

Comment:
What would you think about just deleting the whole namespace here?

Reply:
With Helm3, we're not the ones who create the namespace but yeah deleting the namespace does give a feeling of complete riddance.

Reply:
done